src/components/common/InputField.test.tsx

```diff
@@ -0,0 +1,61 @@
+import { render, screen } from '@/tests/test-utils';
+import userEvent from '@testing-library/user-event';
+import InputField from './InputField';
+
+const FieldProps = {
+  name: 'username',
+  value: '',
+  error: '',
+  isValid: false,
+  onChange: vi.fn(),
+};
+
+describe('공용 InputField 컴포넌트 - 단위 테스트', () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+  });
+
+  it('GIVEN: type prop이 "email"로 주어지면, THEN: input 요소의 타입이 "email"여야 한다.', () => {
+    render(<InputField field={FieldProps} placeholder="이메일" type="email" />);
```

Comment:
이제보니 field라는 props가 아예 큰 덩어리를 받아오는군요.. ㅎㅎ
```suggestion
    render(<InputField {...FieldProps} placeholder="이메일" type="email" />);
```

이렇게 표현될 수 있으면 좋았을 것 같아요!